organization/components/timeslot/TurfCard.tsx

```
@@ -0,0 +1,92 @@
+      <div className="relative h-48 w-full">
+        <Image
+          src={firstImage}
+          alt={turf.name}
+          fill
+          className="object-cover"
+          onError={(e) => {
+            (e.target as HTMLImageElement).src = '/placeholder-turf.jpg';
+          }}
+        />
+        {isSelected && (
+          <div className="absolute inset-0 bg-blue-500/10 "></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)